### PR TITLE
Implementation of Mobile-First Portfolio Using HTML and CSS with Flexbox & Grid box

### PR DESCRIPTION
**Pull request by Mukaba**
## 📱 Implementation of Mobile-First Portfolio Using HTML and CSS with Flexbox & Grid box

**Description:** This pull request update the previous merged pages a two sections📱 designed with mobile-first functionality in mind, using HTML and CSS with Grid and Flexbox 📐. 
The page includes:
-  navigation bar with a logo 📸 ("Logo" by default as in the template) and hamburger menu, 
- a section containing information about the portfolio's owner ✍️, and a list of his contacts 📧.
- a section containing works cards (done projects) about the portfolio's owner ✍️, a list 📧.of languages used by project, and a go live button (see project).
- a section containing languages and skills and a short summary about the portfolio's owner ✍️.

Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,5 +43,134 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
...
+        <button class="see__project">See Project</button>
+    </div>    
+  </section>
+  <section class="about__mySelf">
+    <div class="about__div1">
+        <h2 class="mySelf__title">About <br> Myself</h2>
+    <p class="about_paragraph">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p class="about__links">LET'S CONNECT</p>
```
Comment (Kossi-stack) [inline index.html]: - [ ] The subtitle uses a header tag hx.

File: index.html [hunk 2]
```diff
@@ -43,5 +43,134 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
...
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+    </ul>
+    <button class="get__resume">Get my resume</button>
+    </div>
+    <div class="about__div2">
+        <div class="languages_title">
```
Comment (Kossi-stack) [inline index.html:145]: - [ ] kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li

File: index.html [hunk 3]
```diff
@@ -43,5 +43,134 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
...
+            <p>Languages</p>
+            <div class="lang__dropdown"></div>
+        </div>
+        <div class="languages__pinned flex">
+            <div class="language__view flex">
+                <div class="language__icon js_icon"></div>
```
Comment (Kossi-stack) [inline index.html:151]: - [ ] Kindly make sure the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Mukaba,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Finish Mobile Version PR".

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kamba56 — APPROVED** [review]
Hi @Mukaba,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**